scripts/mirror-images/source-images.yaml

```diff
@@ -0,0 +1,26 @@
+gcr.io:
```

Comment:
To start with, I think this file should have only the tags we have used. Everytime we have to use another tag, we should first update it in source-image.yaml and then once it is mirrored we can use the corresponding tag in the corresponding job.

Reply:
Ack. Updates the source-images.yaml to syncing images with tags that prow-poc is currently using.